### ISSUE TITLE
Combobox role is set on wrapper <div> element instead of <input> element. WCAG complience issue

### ISSUE THREAD

**Issue by TomasVask** [ng-select/ng-select#2426]
### Describe the bug

Combobox role is set on the wrapping `div` element which is not interactive element. This causes the focus of this element to be skipped when navigating with keyboard. There is also `aria-expanded` attribute that is set on `div` element as well.
Was there specific reason for this? Or could it be changed following WCAG example?

### Steps to Reproduce

NA

### Please provide a link to a minimal reproduction of the bug. StackBlitz, CodePen or CodeSanBox

NA

### Expected behaviour

Based on [WCAG documentation](https://www.w3.org/WAI/ARIA/apg/patterns/combobox/examples/combobox-autocomplete-list/) and [MDN](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/combobox_role) combobox role should be set on directly on interactive element in this case `input`. Same applies to `aria-expanded` attribute.

### What version of the library you see this issue?

^12.0.7

### What version of Angular are you using?

Angular 17

### Screenshots

That's what we have in lib's code:
![image](https://github.com/user-attachments/assets/1997b879-d2c3-4269-8449-5292e13b0371)

That's the example from WCAG page:
![image](https://github.com/user-attachments/assets/9526c00f-3bbd-4f69-b195-75f19bb57b3b)

**Comment by mahipureti** [comment]
@TomasVask Can you look at https://github.com/ng-select/ng-select/pull/2428 and see if you can improve?

**Comment by pavankjadda** [comment]
Closed as part #2428
